src/lib/openapi/spec/feature-search-response-schema.ts

```diff
@@ -177,6 +177,30 @@ export const featureSearchResponseSchema = {
                 },
             },
         },
+        createdBy: {
+            type: 'object',
+            description: 'User who created the feature flag',
+            additionalProperties: false,
+            required: ['id', 'name', 'imageUrl'],
```

Comment:
decided to go for nulls instead of null + undefined. All fields will always be provided